torch_harmonics/examples/pde_sphere.py

```diff
@@ -34,6 +34,7 @@
 import torch.nn as nn
 import torch_harmonics as harmonics
 
+import math
 import numpy as np
```

Comment:
is it possible to remove the numpy import here? Ideally we should get rid of numpy for everything that gets imported with torch-harmonics

Reply:
his is in the plotting section, I think numpy is good there since matplotlib is naturally used with numpy